### ISSUE TITLE
Collaborate with existing ongoing efforts :)

### ISSUE THREAD

**Issue by xplshn** [FigSystems/dalixOS#11]
Hi, your goals seem aligned with mine, I would very much love to collaborate, I want to make a minimal system, Unix-y and suckless, but with an interface that doesn't like dogshit, window managers aren't that appealing to me, and I think we could collaborate.

I'm willing to develop GUI programs in Go for this project :)

You can contact me at: https://discord.gg/AgUb9BpwpJ

:)

**Comment by dleggo** [comment]
> Hi, your goals seem aligned with mine, I would very much love to collaborate, I want to make a minimal system, Unix-y and suckless, but with an interface that doesn't like dogshit, window managers aren't that appealing to me, and I think we could collaborate.

I agree that our goals seem quite aligned! Currently, most of our effort is directed toward our application format (IcePak), but once we have IcePak, a WM, a topbar, and a dock, we would love your help with GUI apps! For GUI apps, dalixOS will strive to follow the Apple Human Interface guidlines, and the principles of UI design. https://asktog.com/atc/principles-of-interaction-design/

So unfortunately, it will be a while until we are ready for GUI apps, but we would love your help when we are ready!.
